sourcegraph/cody
# fix chat verical alignment

Follow-up on
- https://github.com/sourcegraph/cody/pull/679

## Test plan
- run Cody for VSCode
- ensure chat input is always aligned to the bottom
